modules/account/src/main/kotlin/edu/stanford/spezi/module/account/manager/AuthenticationManager.kt

```diff
@@ -137,8 +138,7 @@ internal class AuthenticationManager @Inject constructor(
             val googleIdOption = GetGoogleIdOption.Builder()
                 .setFilterByAuthorizedAccounts(filterByAuthorizedAccounts)
                 .setAutoSelectEnabled(true)
-                // TODO: Uncomment once secrets xml has been added in CI secrets
-                // .setServerClientId(context.getString(R.string.serverClientId))
+                .setServerClientId(context.getString(R.string.serverClientId))
```

Comment:
this won't compile locally as we don't have this resource, you need to add the `secrets.xml` file nonetheless with a dummy / empty string. you can also add a comment that get's overriden with the actual id in CI